chat/management/commands/rundiscordbot.py

```diff
@@ -68,6 +39,29 @@ async def send_puzzles_stuck(message):
     await send_puzzles(message, puzzles, "Stuck puzzles")
 
 
+async def send_puzzles_tagged(message):
+    tokens = message.content.split()
+    if len(tokens) < 3:
+        await send_tags(message)
+        return
+    tags = [t.replace("_", " ") for t in tokens[2:]]
+    matched_tags = await sync_to_async(list)(PuzzleTag.objects.filter(name__in=tags))
+    puzzles = await sync_to_async(list)(
+        Puzzle.objects.filter(
+            hunt=settings.BOT_ACTIVE_HUNT,
+            tags__in=matched_tags,
+        ).select_related("chat_room")
+    )
+    await send_puzzles(message, puzzles, f"Puzzles tagged with {tags}")
+
+
+async def send_tags(message):
+    tags = await sync_to_async(list)(PuzzleTag.objects.all())
+    embed = discord.Embed(description="`!puzzles tagged <tag> ...` (use `_` for space)")
```

Comment:
Why is it necessary to use '_' for space? Can't we just treat tokens 3 onward as a single tag?